### PR TITLE
Add support for Windows CE browsers

### PR DESCRIPTION
**Pull request by karpour**
Windows CE remains used to this day on legacy systems, so adding support for these browsers still makes sense. This PR adds support for all versions of Pocket Internet Explorer, which is included in the ROM of pretty much every Windows CE device, as well as the Japanese NetFront browser, which was also popular on Windows CE, among other portable devices and game consoles.

User Agents used for testing:
Microsoft Pocket Internet Explorer/0.6
Mozilla/1.1 (compatible; MSPIE 1.1; Windows CE)
Mozilla/1.1 (compatible; MSPIE 2.0; Windows CE)
Mozilla/2.0 (compatible; MSIE 3.02; Windows CE)
Mozilla/4.0 (compatible; MSIE 4.01; Windows CE)
Mozilla/4.0 (compatible; MSIE 4.01; Windows NT Windows CE)
Mozilla/4.0 (compatible; MSIE 4.01; Windows NT Windows CE; 800x480)
Mozilla/4.0 (compatible; MSIE 5.5; Windows CE)
Mozilla/4.0 (compatible; MSIE 6.0; Windows CE)
Mozilla/2.0 (compatible; MSInternet Explorer 3.02; Windows CE; 240x320)
Mozilla/2.0 (compatible; MSIE 3.02; Windows CE; 240x320)
Mozilla/2.0 (compatible; MSIE 3.02; Windows CE; PPC; javascript; 240x320)
Mozilla/2.0 (compatible; MSIE 3.02; Windows CE; PPC; 240x320)
Mozilla/4.0 (PDA; Windows CE;1.0.0) NetFront/3.0
Mozilla/4.08 (PDA; Windows CE;1.0.0) NetFront/3.1
Mozilla/4.0 (compatible; MSIE 4.01; Windows CE; PPC; 240x320)
Mozilla/2.0 (compatible; MSIE 3.02; Windows CE; Smartphone; javascript; *)
Mozilla/2.0 (compatible; MSIE 3.02; Windows CE; Smartphone; 176x220)
Mozilla/2.0 (compatible; MSIE 3.02; Windows CE; Smartphone; *)
Mozilla/4.0 (compatible; MSIE 4.01; Windows CE; Smartphone; 176x220; SPV C500; OpVer 4.1.1.0)
Mozilla/4.0 (compatible; MSIE 4.01; Windows CE; Smartphone; 176x220; SPV C500; OpVer 4.1.3.0)
Mozilla/4.0 (PDA; Windows CE;0.9.3) NetFront/3.0
Mozilla/4.0 (PDA; Windows CE;1.0.1) NetFront/3.0

### CODE REVIEW

**Comment by DamonOehlman** [comment]
@karpour this looks good to me - happy to add support for those browsers.  Prior to merging this in could you remove the `package-lock.json` file from the PR please?  I think if I'm going to add a lock file into the repo to properly support npm locks, then it should probably be done in a separate PR.

**Comment by karpour** [comment]
Should be good now, package-lock.json is gone.

**Comment by karpour** [comment]
@DamonOehlman I merged with master and resolved conflicts.

**Comment by DamonOehlman** [comment]
@karpour - awesome - merging it in.

**Comment by DamonOehlman** [comment]
And published as `5.3.0`

File: src/index.ts [hunk 1]
```diff
@@ -151,6 +154,9 @@ const userAgentRules: UserAgentRule[] = [
   ['opera-mini', /Opera Mini.*Version\/([0-9\.]+)/],
   ['opera', /Opera\/([0-9\.]+)(?:\s|$)/],
   ['opera', /OPR\/([0-9\.]+)(:?\s|$)/],
+  ['pie',/^Microsoft Pocket Internet Explorer\/(\d+\.\d+)$/],
+  ['pie',/^Mozilla\/\d\.\d+\s\(compatible;\s(?:MSP?IE|MSInternet Explorer) (\d+\.\d+);.*Windows CE.*\)$/],
+  ['netfront',/^Mozilla\/\d\.\d+.*NetFront\/(\d.\d)/],
```
Comment (klarstrup) [inline src/index.ts:159]: missing spaces here

**Pull request merged**